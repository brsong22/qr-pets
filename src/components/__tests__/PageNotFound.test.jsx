
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PageNotFound from '../PageNotFound';

describe('PageNotFound', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<PageNotFound />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
