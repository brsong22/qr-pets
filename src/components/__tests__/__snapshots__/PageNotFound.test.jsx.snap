// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders as expected 1`] = `
<h1>
   Page not found
</h1>
`;
